Template10 (Library)/Controls/HamburgerMenu.xaml.cs

```diff
@@ -502,9 +502,11 @@ public void UpdatePaneMarginToShowHamburgerButton()
             }
         }
 
-        private async Task UpdateSelectedAsync(HamburgerButtonInfo previous, HamburgerButtonInfo value)
+        private bool IsCanceledNavigation;
```

Comment:
Let's change `IsCanceledNavigation` to `DoNotActuallyNavigate`. I realize that the term `IsCanceledNavigation` was tripping me up. I get it now, and your thinking, but it took me a bit and will take me a bit later when I am reviewing for some other reason. The new name speaks very clearly to me and hopefully to other developers who come later. Thanks.